references/apiserver/main/startAPIServer.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+	"io"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/oam-dev/kubevela/pkg/utils/common"
+	"github.com/oam-dev/kubevela/references/apiserver"
+	"github.com/oam-dev/kubevela/references/apiserver/util"
+)
+
+// main will only start up API server
+func main() {
```

Comment:
I put it here, you can use this one: https://github.com/oam-dev/kubevela/blob/master/references/cmd/apiserver/main.go